### PR TITLE
Point README at samvera-vagrant project

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +43,10 @@ We distribute a `docker-compose.yml` configuration for running the Hyku stack an
 docker-compose up -d
 ```
 
+### With Vagrant
+
+The [samvera-vagrant project](https://github.com/samvera-labs/samvera-vagrant) provides another simple way to get started "kicking the tires" of Hyku (and [Hyrax](http://hyr.ax/), making it easy and quick to spin up Hyku. (Note that this is not for production or production-like installations.) It requires VirtualBox and Vagrant.
+
```
Comment (darrenleeweber) [inline README.md]: Might need an extra closing `)` to complete the parens for `(and [Hyrax](http://hyr.ax/))`.

Comment (darrenleeweber) [inline README.md]: Add the links for `VirtualBox` and `Vagrant` too?

Comment (darrenleeweber) [inline README.md]: - https://www.virtualbox.org/
- https://www.vagrantup.com/

**Comment by mjgiarlo** [comment]
@darrenleeweber Done. Thanks!

**Pull request merged**